### ISSUE TITLE
Save and reload a stage position list 

### ISSUE THREAD

**Issue by PatrinaPellett** [oxfordni/support#1]
On occasion, people would like to save multiple points to a list and then revisit these points. Or mark a stage position prior to restarting the software and then revisiting that spot after the hardware has initialized. Currently, the software only allows for one position to be saved and this isn't saved after a software restart.

**Comment by trygub** [comment]
We will add an option in NimOS to add the current stage position to a list of stage positions, which can then be revisited at a later time. A position can be selected and moved to by selecting it from the list.
Would this satisfy your requirements?

**Comment by PatrinaPellett** [comment]
Yes! That sounds great. Additional features to consider for the points list: optimization of the list so that the scope images as fast as possible (i.e. shortest distance between points), the ability to reorder the list, and being able to update/refresh individual points.

**Comment by ollie1400** [comment]
This has been implemented in its basic form and will be present in later software versions.
